shared/xccdf/system/software/integrity.xml

```diff
@@ -98,23 +98,153 @@ must be captured and it should be able to be verified against the installed file
+operating system configurations can have unintended side effects, some of which may
+be relevant to security.
+<br/><br/>
+Detecting such changes and providing an automated response can help avoid unintended,
+negative consequences that could ultimately affect the security state of the operating
+system. The operating system's Information Management Officer (IMO)/Information System
+Security Officer (ISSO) and System Administrators (SAs) must be notified via email and/or
+monitoring system trap when there is an unauthorized modification of a configuration item.
 </rationale>
 <ident prodtype="rhel7" cce="26952-2" />
 <oval id="aide_periodic_cron_checking"/>
 <ref prodtype="rhel7" nist="CM-3(d),CM-3(e),CM-3(5),CM-6(d),CM-6(3),SC-28,SI-7" disa="1744" pcidss="Req-11.5" cis="1.3.1"
-ossrg="SRG-OS-000363-GPOS-00150" sitgid="020150" />
+ossrg="SRG-OS-000363-GPOS-00150" sitgid="020130" />
+</Rule>
+
+<Rule id="aide_scan_notification" severity="medium" prodtype="rhel7">
+<title>Configure Notification of Post-AIDE Scan Details</title>
+<description>
+AIDE should notify appropriate personnel of the details of a scan after the scan has been run.
+If AIDE has already been configured for periodic execution in <tt>/etc/crontab</tt>, append the
+following line to the existing AIDE line:
+<pre> | /bin/mail -s "$(hostname) - AIDE Integrity Check" root@localhost</pre>
+Otherwise, add the following line to <tt>/etc/crontab</tt>:
+<pre>05 4 * * * root /usr/sbin/aide --check | /bin/mail -s "$(hostname) - AIDE Integrity Check" root@localhost</pre>
+AIDE can be executed periodically through other means; this is merely one example.
+</description>
+<ocil clause="AIDE has not been configured or has not been configured to notify personnel of scan details">
+To determine that periodic AIDE execution has been scheduled, run the following command:
+<pre>$ grep aide /etc/crontab</pre>
+The output should return something similar to the following:
+<pre>05 4 * * * root /usr/sbin/aide --check | /bin/mail -s "$(hostname) - AIDE Integrity Check" root@localhost</pre>
+</ocil>
+<rationale>
+Unauthorized changes to the baseline configuration could make the system vulnerable
+to various attacks or allow unauthorized access to the operating system. Changes to
+operating system configurations can have unintended side effects, some of which may
+be relevant to security.
+<br/><br/>
+Detecting such changes and providing an automated response can help avoid unintended,
+negative consequences that could ultimately affect the security state of the operating
+system. The operating system's Information Management Officer (IMO)/Information System
+Security Officer (ISSO) and System Administrators (SAs) must be notified via email and/or
+monitoring system trap when there is an unauthorized modification of a configuration item.
+</rationale>
+<ident prodtype="rhel7" cce="CCE-TBD" />
+<oval id="aide_scan_notification"/>
```

Comment:
Opened https://github.com/OpenSCAP/scap-security-guide/issues/1566 to track OVAL